src/casdoor/main.py

```diff
@@ -70,14 +71,51 @@ def get_oauth_token(self, code: str) -> str:
         :param code: the code that sent from Casdoor using redirect url back to your server.
         :return: access_token
         """
+        r = self.oauth_token_request(code)
+        access_token = r.json().get("access_token")
+
+        return access_token
+
+    def oauth_token_request(self, code: str) -> requests.Response:
```

Comment:
Don't import breaking change

Reply:
> Don't import breaking change

Can your explain where is "breaking change"?

Reply:
No problem now